public/remote.html

```diff
@@ -0,0 +1,94 @@
+
+    const chartHTML = document.getElementById('salarychart')
+
+    const data = salaryJSON.map(salary => salary.salary)
+
+    const labels = data.filter((value, index, self) => self.indexOf(value) === index)
+
+		const remoteTrue = []
+    const remoteFalse = []
+
+    for (let i = 0; i < salaryJSON.length; i++) {
+			if (salaryJSON[i].remote == true) {
+      	remoteTrue.push(salaryJSON[i].salary)
+      } else {
+      	remoteFalse.push(salaryJSON[i].salary)
+      }
+		}
+
+    const frequency = []
+
+    for (let i = 0; i < labels.length; i++) {
```

Comment:
Perhaps this would be easier as:

```
const frequency = labels.map(label =>
  remoteFalse.reduce((n, val) => n + (val === num), 0)
);
```

`.map` has a nice way of cutting boiler plate because you don't have to write the loop yourself, nor extract the array element yourself.  Not that it's hard.  It just cuts out the extra code.

Reply:
Thank for for this! I was trying to figure out how to stick it into a `map`, but I still don't know them well enough. I will test this out.